src/data/community-meetups.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "title": "CryptoCanal",
+    "emoji": ":nl:",
+    "location": "Amsterdam",
+    "link": "https://www.cryptocanal.org/agenda"
+  },
+  {
+    "title": "CryptoCanal",
+    "emoji": ":ch:",
+    "location": "Zurich",
+    "link": "https://www.cryptocanal.org/agenda"
+  },
+  {
```

Comment:
the Netherlands is too small to add the same meetup for multiple cities imo 
Maybe just update location to 'the Netherlands' instead?

Reply:
Sure, can do!

Reply:
I think its fine to add CryptoCanal as 2 location for the Netherlands and Zurich, Swiss.
That makes sense. Someone browsing through the list is looking for meetups in his city/country.

But no need to differentiate between Amsterdam/Rotterdam imo.